ethereum/core/src/main/java/org/hyperledger/besu/ethereum/vm/EVM.java

```diff
@@ -147,7 +148,7 @@ private Operation operationAtOffset(
     final BytesValue bytecode = code.getBytes();
     // If the length of the program code is shorter than the required offset, halt execution.
     if (offset >= bytecode.size()) {
-      return operations.get(STOP_OPCODE, contractAccountVersion);
+      return new VirtualOperation(operations.get(STOP_OPCODE, contractAccountVersion));
```

Comment:
No wrapper please, this will result in too much garbage collection.

Reply:
i can store it once as a field of the class if you prefer. does it make sense ?

Reply:
or put it in the operation registry maybe

Reply:
I think it probably makes sense as a `private final` instance field on `EVM`.